tests/pipeline/resources/parallel-pipeline/config.yaml

```diff
@@ -0,0 +1,12 @@
+topic_name_config:
+  default_error_topic_name: "${component.name}-dead-letter-topic"
+  default_output_topic_name: "${component.name}-test-topic"
+
+schema_registry_url: "http://localhost:8081"
```

Comment:
did you rebase on `v3` or are you going to do that in #312? The config file has changed recently

Reply:
I will probably do it in #312